daft/io/common.py

```diff
@@ -68,7 +69,10 @@ def _get_tabular_files_scan(
             )
 
             scan_op = ScanOperatorHandle.anonymous_scan(
```

Comment:
the two branches here give off some funk, I think it will be much cleaner to have glob be seeded with multiple input paths or chain the streams across different input glob patterns

Reply:
Discussed and made issue in: https://github.com/Eventual-Inc/Daft/pull/1559#discussion_r1383791316